app/views/smart_answers/result.html.erb

```diff
@@ -28,7 +28,16 @@
         } unless outcome.title_as_heading? %>
       <% end %>
 
+      <%= outcome.body %>
+
       <% if @presenter.hide_previous_answers_on_results_page? %>
+        <%= render "govuk_publishing_components/components/heading", {
+          text: "Change your answers",
+          heading_level: 2,
+          font_size: "l",
+          margin_bottom: 6
+        } %>
+
```

Comment:
Could this logic now be moved to the[ _previous_answers partial ](https://github.com/alphagov/smart-answers/blob/eaeded43c367d1436eaa8f31546cf20e48a51114/app/views/smart_answers/_previous_answers.html.erb#L4)instead?

Reply:
Thanks @theseanything - Sure thing great suggestion thanks.  Done.